app/main.go

```diff
@@ -34,6 +35,9 @@ import (
 	"google.golang.org/appengine/log"
 )
 
+// Global Storage Manager object to be used while running the app
+var storageManager storage.Manager
```

Comment:
@knyar So the problem here is that Storage is initialised in the ConfigOptions, not the Config itself:

https://github.com/google/ts-bridge/blob/fe6db8d46812db782dbdb73784943dd11fd983f8/app/main.go#L173-L179

And the config returned in `tsbridge` module is parsed from YAML:
https://github.com/google/ts-bridge/blob/fe6db8d46812db782dbdb73784943dd11fd983f8/tsbridge/config.go#L95

As a result, if I add something to the `Config` struct, it may cause parsing override issues (as you've correctly pointed to me before). As such, I added one package-level var in `main.go` to refer to storage engine.

An alternative approach could be:
- Creating a new config map and method to load it (e.g. `NewRuntimeConfig` and `NewMetricConfig`)
- Leaving it as-is and adressing more thoughtfully when we address the env variables vs. YAML config

But again, I'm new to Go - so if I'm missing anything or you have another approach in mind - just let me know! :)

Reply:
Agreed, rolled it into a helper function.